tools/Environments/DevHome.Environments/ViewModels/LandingPageViewModel.cs

```diff
@@ -151,22 +244,60 @@ public void SearchHandler(string query)
         };
     }
 
+    /// <summary>
+    /// Updates the view model to filter the compute systems according to the provider.
+    /// </summary>
+    [RelayCommand]
+    public void ProviderHandler()
+    {
+        var currentProvider = Providers[SelectedProviderIndex];
+        ComputeSystemsView.Filter = system =>
+        {
+            if (currentProvider == _stringResource.GetLocalized("AllProviders"))

```

Comment:
Please use `.Equals` to compare strings.

Reply:
Changed.